roles/openshift_project_request_template/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+- when: openshift_project_request_template_manage
+  block:
+    - name: Create temp file for template
+      command: mktemp /tmp/openshift-ansible-XXXXXX.yaml
```

Comment:
Will there be anything sensitive in this file?  If so, we should make it somewhere other than /tmp.

Reply:
`mktemp` creates the file with mode `0600`, and in the vast majority of cases I don't expect sensitive data to be in this file--the resources contained are copied to all newly created projects.